src/main/java/com/puppycrawl/tools/checkstyle/utils/ScopeUtils.java

```diff
@@ -209,7 +209,8 @@ public static boolean isInCodeBlock(DetailAST node) {
             if (type == TokenTypes.METHOD_DEF
                 || type == TokenTypes.CTOR_DEF
                 || type == TokenTypes.INSTANCE_INIT
-                || type == TokenTypes.STATIC_INIT) {
+                || type == TokenTypes.STATIC_INIT
+                || type == TokenTypes.LAMBDA) {
```

Comment:
Why lambda and not field definition? Are all lambdas considered a code block from the perspective of this utility?
There is no removal of violation in report for this which is still a lambda: https://mezk.github.io/i1254-JavadocVariable/xref/home/aselkin/IDEA_Projects/contribution/checkstyle-tester/src/main/java/openjdk8/src/share/classes/java/time/temporal/TemporalQueries.java.html#L381
Makes me think lambda isn't the direct issue.

Either way, how about we add the change to the method javadoc? It wasn't updated.